### ISSUE TITLE
Version 1.2.0 broke features from version 1.1.6

### ISSUE THREAD

**Issue by SaraFarsi** [grafana/piechart-panel#125]
When I updated piechart-panel from 1.1.6 to 1.2.0, the scroll bar broke and the size of the pie charts decreased. Is this a bug?!

**Comment by marefr** [comment]
For the 1.2.0 release we did some fixes for Grafana v5.0. We haven't got any similar feedback as this. What Grafana version are you using?

**Comment by SaraFarsi** [comment]
we are using Grafana v4.6.3 since I believe 5.0.0 is not released yet!

**Comment by raidsec** [comment]
Same problem here. pie chart is always buggy and on update you destroy your current dashboard. sad...

**Comment by raidsec** [comment]
For me i have problems with the "under graph" information. It looks like that you always have to scroll and the circle will not be reduced to fit into the box and fully show the informations below:

![issue_pie_1](https://user-images.githubusercontent.com/33294982/37039522-ff4c8b02-2157-11e8-96f8-67bf88684305.png)

You always have to scroll. 

![issue_pie_2](https://user-images.githubusercontent.com/33294982/37039524-ff69041c-2157-11e8-9498-6fb9b41f4fdd.png)

I use Google Chrome 63.0.3239.132 64bit. I already cleared the browser cache and restarted my machine.

**Comment by Perlovka** [comment]
With legend position "Right side" scroll bar is absent.

**Comment by raidsec** [comment]
Thats correct but "right side" legend needs more space then "under". I use 4 pie chats in one row.

**Comment by raidsec** [comment]
hello, is there any update here?
